### DISCUSSION TITLE
rustdesk flutter web

### DISCUSSION THREAD

**Discussion by karthikindia** [rustdesk/rustdesk#946] [Q&A]
Dear All
Good day to the community
Any one build the rustdesk web - https://github.com/rustdesk/rustdesk/tree/master/flutter/web and built successful
please share your experiences here.
I am having difficulty in building that

**Comment by TBG-FR** [comment]
Did you manage to do it ? We should be able to build the web client from the current sources, but that requires a bit of effort... 

🔗  #2535 https://github.com/rustdesk/rustdesk/issues/4579#issuecomment-1581971865 #3742 etc.
